api/controllers/EAppReferenceController.js

```diff
@@ -0,0 +1,29 @@
+const sails = require('sails');
+
+const EAppReferenceController = {
+    renderPage(req, res) {
+        const userData = HelperService.getUserData(req, res);
+        if (!userData.loggedIn) {
+            sails.log.error('User not logged in');
+            return res.serverError();
```

Comment:
Good point. It's quite interesting, the sign-in is handled by a completely different repo so it would have been nice to redirect to the login page with a message saying 'you don't have access to this page, but that ability doesn't exist :(

The res.forbidden() might be the next best thing, but I don't think that's configured to render a forbidden page 🤔

Reply:
Actually, looks like forbidden redirects to the 403 page:

https://github.com/UKForeignOffice/loi-application-service/blob/develop/api/responses/forbidden.js#L29 

which... has the same text as the 500 and 404 pages 🤦 

Ah well 🤷